UnitsNet.Tests/CustomCode/FlowTests.cs

```diff
@@ -27,6 +27,8 @@ public class FlowTests : FlowTestsBase
 
         protected override double CubicDecimetersPerMinuteInOneCubicMeterPerSecond => 60000.00000;
 
+        protected override double CubicFeetPerHourInOneCubicMeterPerSecond => 1.2713300937959091422759633856933e+5;
```

Comment:
This value is slightly different than CubicFeetPerSecond multiplied by 3600.

`35.314666213 * 3600 = 1.271327983668e+5`

Which one should be corrected?

Reply:
Since CubicFeetPerSecond was already there, I would use `1.271327983668e+5` as a test value for CubicFeetPerHour

Reply:
I confirmed with two online converters cubic feet per second:

`35.314666721489 ft3/s` from [converter1](https://www.unitjuggler.com/convert-flowrate-from-m3s-to-cfm.html)
`35.314 666 721 ft3/s` from [converter2](http://www.traditionaloven.com/tutorials/flow-rate/convert-m3-cubic-meter-per-second-to-ft3-cubic-foot-per-second.html)

The value is not perfectly identical, but pretty much good.

So, using the first value above times 3600, the per hour value should be: `127132.8001973604`.

If you could update both foot per second and foot per hour accordingly, then this should be good.